src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,102 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * The Storage object deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+    private final String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Writes the current list of tasks to the duke.txt file.
+     *
+     * @param tasks TaskList containing current list of tasks.
+     * @throws DukeException If unable to create or write file.
+     */
+    public void write(TaskList tasks) throws DukeException {
```

Comment:
Perhaps giving more details to the method name would be better, such as writeToFile, etc...